content/en/tracing/faq/php-tracer-manual-installation.md

```diff
@@ -5,11 +5,39 @@ kind: faq
 
 ## Before you start
 
-Datadog strongly encourages you to install the PHP extension through the [pre-built packages][1]. If you still prefer to manually install the PHP tracer, follow the steps below.
+Datadog strongly encourages you to install the PHP extension through the [official installer][1]. If you still prefer to use alternative installation methods, follow the steps below.
+
+<div class="alert alert-info">
+<strong>Note:</strong>
+It might take a few minutes before traces appear in the UI. If traces still do not appear after a few minutes, create a <a href="/tracing/troubleshooting/tracer_startup_logs?tab=php#php-info"><code>phpinfo()</code></a> page from the host machine and scroll down to the "ddtrace" section. Failed diagnostic checks will appear here to help identify any issues.
```

Comment:
```suggestion
It may take a few minutes before traces appear in the UI. If traces do not appear after a few minutes, create a <a href="/tracing/troubleshooting/tracer_startup_logs?tab=php#php-info"><code>phpinfo()</code></a> page from the host machine and scroll down to `ddtrace`. Failed diagnostic checks appear in this section to help identify any issues.
```